<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gboxOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QPushButton" name="btnNewVol">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Volume</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinNewVol">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New Volume Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Volume Value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOldVol">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoUseDefDevice">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoUseCustDevice">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmdDeviceList">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gboxInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>641</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QPushButton" name="btnNewVol_In">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Volume</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinNewVol_In">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New Volume Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Volume Value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOldVol_In">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoUseDefDevice_In">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cmdDeviceList_In">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoUseCustDevice_In">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
